plugins/package-managers/gradle-inspector/src/main/kotlin/GradleInspector.kt

```diff
@@ -168,9 +180,16 @@ class GradleInspector(
                         addProgressListener(ProgressListener { logger.debug(it.displayName) })
                     }
 
-                    options[OPTION_JAVA_HOME]?.also {
+                    val javaHome = options[OPTION_JAVA_VERSION]?.let {
+                        JavaBootstrapper.installJdk("TEMURIN", it)
+                            .onFailure { e ->
+                                issues += createAndLogIssue(managerName, e.collectMessages())
+                            }.getOrNull()
+                    } ?: options[OPTION_JAVA_HOME]?.let { File(it) }
```

Comment:
The docs say `javaHome` overrides `javaVersion` but it's implemented the other way round.

Reply:
I've adjusted the docs.